common/djangoapps/adaptive_learning/config/tests/utils.py

```diff
@@ -17,6 +17,10 @@ def adaptive_learning_enabled_feature_flags(
         course_id=None,
         enabled_for_course=False
 ):
+    """
+    Sets the global setting and the course-specific
+    setting (if the course_id is given) for AdaptiveLearning.
```

Comment:
@tomaszgy Thanks for adding this docstring! Just a small typo: "AdaptiveLearning" should be "Adaptive Learning".

Reply:
👍